node/index.js

```diff
@@ -418,9 +418,7 @@ function TChannelConnection(channel, socket, direction, remoteAddr) {
 
     // TODO: refactor handler to be objectMode Writable
     self.reader.on('data', function onReaderFrame(frame) {
-        if (!self.closing) {
```

Comment:
we should add the `socket.destroy()` to resetAll() before we remove these checks.